1_Mechanics/02_GMGuide.md

```diff
@@ -1,142 +1,1052 @@
+allowing a Boss to recover a small amount of HP or gain Fate Points. They may
+allow for an additional attack, or an environmental effect involving a Check
+for the Players. Players who fail might take damage or receive a Status
+Condition like Entangled. 
+
+For example, a poisonous monster would have the following Adversary Action: 
+*Acid Splash*. A cloud of acid appears in a 4 space cube. Any creature inside
+ the cube at the start of their turn makes a DR 3 Agility Check or take 2
+ damage. On a success, 1 damage.
+
+#### House Actions
+
+House Actions take place when a Boss is in their home or base of operations.
+Once initiated, they happen at the beginning of every round of combat. This
+is a great way to add in dynamic elements to an encounter, such as new
+enemies appearing or the environment becoming more dangerous.
+
+#### Boss Phases
+
+Dealers often have < 10 HP. For balance, Boss Encounters can have multiple
+Phases. After running out of HP, they replenish and changes their approach.
+Every time a Boss enters a new Phase, draw a new TC for them. The following
+list features a number of examples that might be triggered by a new Phase:
+
+- Boss escapes to another location 
+- Boss summons Minions 
+- Boss alters the environment to make certain areas 
+    + cause damage (e.g., erupting flame)
+    + require a check (e.g,. Agility check to stay standing on ice) 
+- Boss disappears, and reappears
+    + when Minions have been defeated 
+    + after a set number of rounds 
+    + after an environmental effect dissipates 
+- Boss changes features like
+    + core stats
+    + higher or lower AR
+    + drawing with Upper Hand (e.g., putting in more effort)
+    + drawing with Lower Hand (e.g., getting angry/sloppy)
+
+### Example Combat Encounter designed for 4 Players at Level 2
+
+In a longer campaign, mix up the types of encounters to see how Players react
+and interact with each other. This will keep things fresh so combat keeps
+Players engaged over time. 
+Here, the context is kept purposefully blank to fill in with your setting.
+
+**Location.** Start with a wide open space for the Players and enemies to move
+  around in, between 10-20 spaces across to allow for movement and let the
+  range influence Player tactics. 
+
+**Setup.** Add 4 Minions close to the entrance and 1 Dealer farther away. 2
+  Minions are melee guards and 2 are shooters 3-5 spaces away, to force the
+  Players to choose their targets and allow space between each enemy. The
+  Players will need to deal with the Minions first in order to reach the
+  Dealer, allowing the Dealer to use their abilities in the first 2+ rounds. 
+
+#### Stat Blocks
+
+Every creature has a set of statistics that govern their actions in the game. Consult their stat block to learn their strengths and weaknesses, and what actions they can take in combat.
+
+If you want to make your own custom creature, you can use the sample template provided below: 
+
+##### Name
```

Comment:
Examples below are bold entity name. Change to reflect same here?
```suggestion
**Name**
```

Reply:
I think we can leave it as ##### h5 as originally stated, sorry I didn't update that in the last pass. Making a commit to update that now - I think we should stick with h5 as a naming convention in other docs for NPCs or stat blocks so we can render these specifically uniquely in the HTML/Website render side of things rather than bolding it.